portfolio/src/main/webapp/style.css

```diff
@@ -66,11 +101,33 @@ h3 {
   width: 50%;
 }
 
-p {
+p{
   color: #242d49;
   font-family: 'Montserrat', sans-serif;
 }
 
+.font-style{
+  font-family: 'Montserrat', sans-serif;
+}
+
+.box-shadow{
+  border-radius: 10px;
+  box-shadow: rgba(149, 157, 165, 0.2) 0px 8px 24px;
+}
+
+.margin-div{
+  padding: 5em;
+  margin: 5em;
+}
+
+.margin-left{
+  margin-left: 10px;
+}
+
+.margin-top{
```

Comment:
[4.2.5](https://google.github.io/styleguide/htmlcssguide.html#Property_Name_Stops): Use a space between the last selector and the declaration block.

Reply:
This issue also exists in some of the other places I wasn't able to comment.